pkg/apis/upgrade.cattle.io/v1/types.go

```diff
@@ -44,6 +44,8 @@ type PlanSpec struct {
 
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
 
+	OnePlanPerNode bool `json:"onePlanPerNode,omitempty"`
```

Comment:
is `Exclusive` perhaps a better name for this option? `OnePlanPerNode` is a bit of a mouthful.

Reply:
I agree, I do like `Exclusive`, I think that is concise.

Reply:
Name changed.